pkgs/development/compilers/ghc/common-hadrian.nix

```diff
@@ -285,7 +285,7 @@ let
     # https://gitlab.haskell.org/ghc/ghc/-/issues/22081
     ++ lib.optional enableDwarf elfutils
     ++ lib.optional (!enableNativeBignum) gmp
-    ++ lib.optional (platform.libc != "glibc" && !targetPlatform.isWindows && !targetPlatform.isGhcjs) libiconv;
+    ++ lib.optional (platform.libc != "glibc" && !targetPlatform.isWindows && !targetPlatform.isGhcjs && !targetPlatform.useAndroidPrebuilt) libiconv;
```

Comment:
Otherwise
```
$ nix-build -A pkgsCross.aarch64-android-prebuilt.haskell.packages.ghc910.primitive
...
aarch64-unknown-linux-android-ld: error: attempted static link of dynamic object /nix/store/dc4sk70a32x7qzvqr81rq9036cslrlw3-libiconv-aarch64-unknown-linux-android-1.17/lib/libiconv.so
<no location info>: error:
    `aarch64-unknown-linux-android-ld' failed in phase `Merge objects'. (Exit code: 1)
```

Reply:
These errors have to be in the commit msg

Reply:
Fixed, I think